### DISCUSSION TITLE
Was my account deleted?

### DISCUSSION THREAD

**Discussion by roke-julian-lockhart** [bookwyrm-social/bookwyrm#3012] [Q&A] [answered]
Per https://mastodon.social/@rokejulianlockhart@bookwyrm.social, I registered the account @rokejulianlockhart@bookwyrm.social about a year ago. However, for the last 6 or so months, I've been entirely unable to login, because my username and password aren't recognized. Trying to get a password reset sent to my registered e-mail address simply doesn't work. Additionally, visiting https://bookwyrm.social/user/rokejulianlockhart gives me

> The page you requested doesn't seem to exist!

Was my account deleted somehow?! I had a lot of data in it.

**Comment by mouse-reeve** [comment, marked as answer]
We resolved this by email ❤️
